### PR TITLE
Use tabulate in place of libtbx.table_utils

### PR DESCRIPTION
**Pull request by rjgildea**
- Replace libtbx.table_utils.simple_table with [tabulate](https://pypi.org/project/tabulate/)
- Replace libtbx.table_utils.format with [tabulate](https://pypi.org/project/tabulate/)

This PR currently `uses tablefmt="rst"`. Alternatives table formats consist of:
```
from tabulate import tabulate_formats
print(tabulate_formats)
[u'fancy_grid', u'github', u'grid', u'html', u'jira', u'latex', u'latex_booktabs', u'latex_raw', u'mediawiki', u'moinmoin', u'orgtbl', u'pipe', u'plain', u'presto', u'psql', u'rst', u'simple', u'textile', u'tsv', u'youtrack']
```

Comparison of the different table formats using the `dials.refine` "Refinement steps" table as an example:
:bikeshed:
```
from tabulate import tabulate_formats
for tablefmt in tabulate_formats:
    print(tablefmt)
    print(tabulate(rows, header, tablefmt=tablefmt))

fancy_grid
╒════════╤════════╤══════════╤══════════╤════════════╕
│   Step │   Nref │   RMSD_X │   RMSD_Y │   RMSD_Phi │
│        │        │     (mm) │     (mm) │      (deg) │
╞════════╪════════╪══════════╪══════════╪════════════╡
│      0 │  89562 │ 0.046773 │ 0.043273 │   0.021281 │
├────────┼────────┼──────────┼──────────┼────────────┤
│      1 │  89562 │ 0.046872 │ 0.042125 │   0.021623 │
├────────┼────────┼──────────┼──────────┼────────────┤
│      2 │  89562 │ 0.04684  │ 0.042104 │   0.021646 │
├────────┼────────┼──────────┼──────────┼────────────┤
│      3 │  89562 │ 0.046778 │ 0.042111 │   0.021662 │
├────────┼────────┼──────────┼──────────┼────────────┤
│      4 │  89562 │ 0.046719 │ 0.042129 │   0.02167  │
├────────┼────────┼──────────┼──────────┼────────────┤
│      5 │  89562 │ 0.04666  │ 0.042168 │   0.02167  │
├────────┼────────┼──────────┼──────────┼────────────┤
│      6 │  89562 │ 0.046622 │ 0.042203 │   0.021666 │
├────────┼────────┼──────────┼──────────┼────────────┤
│      7 │  89562 │ 0.04661  │ 0.042215 │   0.021664 │
├────────┼────────┼──────────┼──────────┼────────────┤
│      8 │  89562 │ 0.046609 │ 0.042216 │   0.021664 │
╘════════╧════════╧══════════╧══════════╧════════════╛
github
|   Step |   Nref |          RMSD_X
(mm) |          RMSD_Y
(mm) |          RMSD_Phi
(deg) |
|--------|--------|----------|----------|----------|
|      0 |  89562 | 0.046773 | 0.043273 | 0.021281 |
|      1 |  89562 | 0.046872 | 0.042125 | 0.021623 |
|      2 |  89562 | 0.04684  | 0.042104 | 0.021646 |
|      3 |  89562 | 0.046778 | 0.042111 | 0.021662 |
|      4 |  89562 | 0.046719 | 0.042129 | 0.02167  |
|      5 |  89562 | 0.04666  | 0.042168 | 0.02167  |
|      6 |  89562 | 0.046622 | 0.042203 | 0.021666 |
|      7 |  89562 | 0.04661  | 0.042215 | 0.021664 |
|      8 |  89562 | 0.046609 | 0.042216 | 0.021664 |
grid
+--------+--------+----------+----------+------------+
|   Step |   Nref |   RMSD_X |   RMSD_Y |   RMSD_Phi |
|        |        |     (mm) |     (mm) |      (deg) |
+========+========+==========+==========+============+
|      0 |  89562 | 0.046773 | 0.043273 |   0.021281 |
+--------+--------+----------+----------+------------+
|      1 |  89562 | 0.046872 | 0.042125 |   0.021623 |
+--------+--------+----------+----------+------------+
|      2 |  89562 | 0.04684  | 0.042104 |   0.021646 |
+--------+--------+----------+----------+------------+
|      3 |  89562 | 0.046778 | 0.042111 |   0.021662 |
+--------+--------+----------+----------+------------+
|      4 |  89562 | 0.046719 | 0.042129 |   0.02167  |
+--------+--------+----------+----------+------------+
|      5 |  89562 | 0.04666  | 0.042168 |   0.02167  |
+--------+--------+----------+----------+------------+
|      6 |  89562 | 0.046622 | 0.042203 |   0.021666 |
+--------+--------+----------+----------+------------+
|      7 |  89562 | 0.04661  | 0.042215 |   0.021664 |
+--------+--------+----------+----------+------------+
|      8 |  89562 | 0.046609 | 0.042216 |   0.021664 |
+--------+--------+----------+----------+------------+
html
<table>
<thead>
<tr><th style="text-align: right;">  Step</th><th style="text-align: right;">  Nref</th><th style="text-align: right;">         RMSD_X
(mm)</th><th style="text-align: right;">         RMSD_Y
(mm)</th><th style="text-align: right;">         RMSD_Phi
(deg)</th></tr>
</thead>
<tbody>
<tr><td style="text-align: right;">     0</td><td style="text-align: right;"> 89562</td><td style="text-align: right;">0.046773</td><td style="text-align: right;">0.043273</td><td style="text-align: right;">0.021281</td></tr>
<tr><td style="text-align: right;">     1</td><td style="text-align: right;"> 89562</td><td style="text-align: right;">0.046872</td><td style="text-align: right;">0.042125</td><td style="text-align: right;">0.021623</td></tr>
<tr><td style="text-align: right;">     2</td><td style="text-align: right;"> 89562</td><td style="text-align: right;">0.04684 </td><td style="text-align: right;">0.042104</td><td style="text-align: right;">0.021646</td></tr>
<tr><td style="text-align: right;">     3</td><td style="text-align: right;"> 89562</td><td style="text-align: right;">0.046778</td><td style="text-align: right;">0.042111</td><td style="text-align: right;">0.021662</td></tr>
<tr><td style="text-align: right;">     4</td><td style="text-align: right;"> 89562</td><td style="text-align: right;">0.046719</td><td style="text-align: right;">0.042129</td><td style="text-align: right;">0.02167 </td></tr>
<tr><td style="text-align: right;">     5</td><td style="text-align: right;"> 89562</td><td style="text-align: right;">0.04666 </td><td style="text-align: right;">0.042168</td><td style="text-align: right;">0.02167 </td></tr>
<tr><td style="text-align: right;">     6</td><td style="text-align: right;"> 89562</td><td style="text-align: right;">0.046622</td><td style="text-align: right;">0.042203</td><td style="text-align: right;">0.021666</td></tr>
<tr><td style="text-align: right;">     7</td><td style="text-align: right;"> 89562</td><td style="text-align: right;">0.04661 </td><td style="text-align: right;">0.042215</td><td style="text-align: right;">0.021664</td></tr>
<tr><td style="text-align: right;">     8</td><td style="text-align: right;"> 89562</td><td style="text-align: right;">0.046609</td><td style="text-align: right;">0.042216</td><td style="text-align: right;">0.021664</td></tr>
</tbody>
</table>
jira
||   Step ||   Nref ||   RMSD_X ||   RMSD_Y ||   RMSD_Phi ||
||        ||        ||     (mm) ||     (mm) ||      (deg) ||
|      0 |  89562 | 0.046773 | 0.043273 |   0.021281 |
|      1 |  89562 | 0.046872 | 0.042125 |   0.021623 |
|      2 |  89562 | 0.04684  | 0.042104 |   0.021646 |
|      3 |  89562 | 0.046778 | 0.042111 |   0.021662 |
|      4 |  89562 | 0.046719 | 0.042129 |   0.02167  |
|      5 |  89562 | 0.04666  | 0.042168 |   0.02167  |
|      6 |  89562 | 0.046622 | 0.042203 |   0.021666 |
|      7 |  89562 | 0.04661  | 0.042215 |   0.021664 |
|      8 |  89562 | 0.046609 | 0.042216 |   0.021664 |
latex
\begin{tabular}{rrrrr}
\hline
   Step &   Nref &          RMSD\_X
(mm) &          RMSD\_Y
(mm) &          RMSD\_Phi
(deg) \\
\hline
      0 &  89562 & 0.046773 & 0.043273 & 0.021281 \\
      1 &  89562 & 0.046872 & 0.042125 & 0.021623 \\
      2 &  89562 & 0.04684  & 0.042104 & 0.021646 \\
      3 &  89562 & 0.046778 & 0.042111 & 0.021662 \\
      4 &  89562 & 0.046719 & 0.042129 & 0.02167  \\
      5 &  89562 & 0.04666  & 0.042168 & 0.02167  \\
      6 &  89562 & 0.046622 & 0.042203 & 0.021666 \\
      7 &  89562 & 0.04661  & 0.042215 & 0.021664 \\
      8 &  89562 & 0.046609 & 0.042216 & 0.021664 \\
\hline
\end{tabular}
latex_booktabs
\begin{tabular}{rrrrr}
\toprule
   Step &   Nref &          RMSD\_X
(mm) &          RMSD\_Y
(mm) &          RMSD\_Phi
(deg) \\
\midrule
      0 &  89562 & 0.046773 & 0.043273 & 0.021281 \\
      1 &  89562 & 0.046872 & 0.042125 & 0.021623 \\
      2 &  89562 & 0.04684  & 0.042104 & 0.021646 \\
      3 &  89562 & 0.046778 & 0.042111 & 0.021662 \\
      4 &  89562 & 0.046719 & 0.042129 & 0.02167  \\
      5 &  89562 & 0.04666  & 0.042168 & 0.02167  \\
      6 &  89562 & 0.046622 & 0.042203 & 0.021666 \\
      7 &  89562 & 0.04661  & 0.042215 & 0.021664 \\
      8 &  89562 & 0.046609 & 0.042216 & 0.021664 \\
\bottomrule
\end{tabular}
latex_raw
\begin{tabular}{rrrrr}
\hline
   Step &   Nref &          RMSD_X
(mm) &          RMSD_Y
(mm) &          RMSD_Phi
(deg) \\
\hline
      0 &  89562 & 0.046773 & 0.043273 & 0.021281 \\
      1 &  89562 & 0.046872 & 0.042125 & 0.021623 \\
      2 &  89562 & 0.04684  & 0.042104 & 0.021646 \\
      3 &  89562 & 0.046778 & 0.042111 & 0.021662 \\
      4 &  89562 & 0.046719 & 0.042129 & 0.02167  \\
      5 &  89562 & 0.04666  & 0.042168 & 0.02167  \\
      6 &  89562 & 0.046622 & 0.042203 & 0.021666 \\
      7 &  89562 & 0.04661  & 0.042215 & 0.021664 \\
      8 &  89562 & 0.046609 & 0.042216 & 0.021664 \\
\hline
\end{tabular}
mediawiki
{| class="wikitable" style="text-align: left;"
|+ <!-- caption -->
|-
! align="right"|   Step !! align="right"|   Nref !! align="right"|          RMSD_X
(mm) !! align="right"|          RMSD_Y
(mm) !! align="right"|          RMSD_Phi
(deg)
|-
| align="right"|      0 || align="right"|  89562 || align="right"| 0.046773 || align="right"| 0.043273 || align="right"| 0.021281
|-
| align="right"|      1 || align="right"|  89562 || align="right"| 0.046872 || align="right"| 0.042125 || align="right"| 0.021623
|-
| align="right"|      2 || align="right"|  89562 || align="right"| 0.04684  || align="right"| 0.042104 || align="right"| 0.021646
|-
| align="right"|      3 || align="right"|  89562 || align="right"| 0.046778 || align="right"| 0.042111 || align="right"| 0.021662
|-
| align="right"|      4 || align="right"|  89562 || align="right"| 0.046719 || align="right"| 0.042129 || align="right"| 0.02167
|-
| align="right"|      5 || align="right"|  89562 || align="right"| 0.04666  || align="right"| 0.042168 || align="right"| 0.02167
|-
| align="right"|      6 || align="right"|  89562 || align="right"| 0.046622 || align="right"| 0.042203 || align="right"| 0.021666
|-
| align="right"|      7 || align="right"|  89562 || align="right"| 0.04661  || align="right"| 0.042215 || align="right"| 0.021664
|-
| align="right"|      8 || align="right"|  89562 || align="right"| 0.046609 || align="right"| 0.042216 || align="right"| 0.021664
|}
moinmoin
||<style="text-align: right;"> '''   Step ''' ||<style="text-align: right;"> '''   Nref ''' ||<style="text-align: right;"> '''          RMSD_X
(mm) ''' ||<style="text-align: right;"> '''          RMSD_Y
(mm) ''' ||<style="text-align: right;"> '''          RMSD_Phi
(deg) ''' ||
||<style="text-align: right;">       0  ||<style="text-align: right;">   89562  ||<style="text-align: right;">  0.046773  ||<style="text-align: right;">  0.043273  ||<style="text-align: right;">  0.021281  ||
||<style="text-align: right;">       1  ||<style="text-align: right;">   89562  ||<style="text-align: right;">  0.046872  ||<style="text-align: right;">  0.042125  ||<style="text-align: right;">  0.021623  ||
||<style="text-align: right;">       2  ||<style="text-align: right;">   89562  ||<style="text-align: right;">  0.04684   ||<style="text-align: right;">  0.042104  ||<style="text-align: right;">  0.021646  ||
||<style="text-align: right;">       3  ||<style="text-align: right;">   89562  ||<style="text-align: right;">  0.046778  ||<style="text-align: right;">  0.042111  ||<style="text-align: right;">  0.021662  ||
||<style="text-align: right;">       4  ||<style="text-align: right;">   89562  ||<style="text-align: right;">  0.046719  ||<style="text-align: right;">  0.042129  ||<style="text-align: right;">  0.02167   ||
||<style="text-align: right;">       5  ||<style="text-align: right;">   89562  ||<style="text-align: right;">  0.04666   ||<style="text-align: right;">  0.042168  ||<style="text-align: right;">  0.02167   ||
||<style="text-align: right;">       6  ||<style="text-align: right;">   89562  ||<style="text-align: right;">  0.046622  ||<style="text-align: right;">  0.042203  ||<style="text-align: right;">  0.021666  ||
||<style="text-align: right;">       7  ||<style="text-align: right;">   89562  ||<style="text-align: right;">  0.04661   ||<style="text-align: right;">  0.042215  ||<style="text-align: right;">  0.021664  ||
||<style="text-align: right;">       8  ||<style="text-align: right;">   89562  ||<style="text-align: right;">  0.046609  ||<style="text-align: right;">  0.042216  ||<style="text-align: right;">  0.021664  ||
orgtbl
|   Step |   Nref |   RMSD_X |   RMSD_Y |   RMSD_Phi |
|        |        |     (mm) |     (mm) |      (deg) |
|--------+--------+----------+----------+------------|
|      0 |  89562 | 0.046773 | 0.043273 |   0.021281 |
|      1 |  89562 | 0.046872 | 0.042125 |   0.021623 |
|      2 |  89562 | 0.04684  | 0.042104 |   0.021646 |
|      3 |  89562 | 0.046778 | 0.042111 |   0.021662 |
|      4 |  89562 | 0.046719 | 0.042129 |   0.02167  |
|      5 |  89562 | 0.04666  | 0.042168 |   0.02167  |
|      6 |  89562 | 0.046622 | 0.042203 |   0.021666 |
|      7 |  89562 | 0.04661  | 0.042215 |   0.021664 |
|      8 |  89562 | 0.046609 | 0.042216 |   0.021664 |
pipe
|   Step |   Nref |   RMSD_X |   RMSD_Y |   RMSD_Phi |
|        |        |     (mm) |     (mm) |      (deg) |
|-------:|-------:|---------:|---------:|-----------:|
|      0 |  89562 | 0.046773 | 0.043273 |   0.021281 |
|      1 |  89562 | 0.046872 | 0.042125 |   0.021623 |
|      2 |  89562 | 0.04684  | 0.042104 |   0.021646 |
|      3 |  89562 | 0.046778 | 0.042111 |   0.021662 |
|      4 |  89562 | 0.046719 | 0.042129 |   0.02167  |
|      5 |  89562 | 0.04666  | 0.042168 |   0.02167  |
|      6 |  89562 | 0.046622 | 0.042203 |   0.021666 |
|      7 |  89562 | 0.04661  | 0.042215 |   0.021664 |
|      8 |  89562 | 0.046609 | 0.042216 |   0.021664 |
plain
  Step    Nref    RMSD_X    RMSD_Y    RMSD_Phi
                    (mm)      (mm)       (deg)
     0   89562  0.046773  0.043273    0.021281
     1   89562  0.046872  0.042125    0.021623
     2   89562  0.04684   0.042104    0.021646
     3   89562  0.046778  0.042111    0.021662
     4   89562  0.046719  0.042129    0.02167
     5   89562  0.04666   0.042168    0.02167
     6   89562  0.046622  0.042203    0.021666
     7   89562  0.04661   0.042215    0.021664
     8   89562  0.046609  0.042216    0.021664
presto
   Step |   Nref |   RMSD_X |   RMSD_Y |   RMSD_Phi
        |        |     (mm) |     (mm) |      (deg)
--------+--------+----------+----------+------------
      0 |  89562 | 0.046773 | 0.043273 |   0.021281
      1 |  89562 | 0.046872 | 0.042125 |   0.021623
      2 |  89562 | 0.04684  | 0.042104 |   0.021646
      3 |  89562 | 0.046778 | 0.042111 |   0.021662
      4 |  89562 | 0.046719 | 0.042129 |   0.02167
      5 |  89562 | 0.04666  | 0.042168 |   0.02167
      6 |  89562 | 0.046622 | 0.042203 |   0.021666
      7 |  89562 | 0.04661  | 0.042215 |   0.021664
      8 |  89562 | 0.046609 | 0.042216 |   0.021664
psql
+--------+--------+----------+----------+------------+
|   Step |   Nref |   RMSD_X |   RMSD_Y |   RMSD_Phi |
|        |        |     (mm) |     (mm) |      (deg) |
|--------+--------+----------+----------+------------|
|      0 |  89562 | 0.046773 | 0.043273 |   0.021281 |
|      1 |  89562 | 0.046872 | 0.042125 |   0.021623 |
|      2 |  89562 | 0.04684  | 0.042104 |   0.021646 |
|      3 |  89562 | 0.046778 | 0.042111 |   0.021662 |
|      4 |  89562 | 0.046719 | 0.042129 |   0.02167  |
|      5 |  89562 | 0.04666  | 0.042168 |   0.02167  |
|      6 |  89562 | 0.046622 | 0.042203 |   0.021666 |
|      7 |  89562 | 0.04661  | 0.042215 |   0.021664 |
|      8 |  89562 | 0.046609 | 0.042216 |   0.021664 |
+--------+--------+----------+----------+------------+
rst
======  ======  ========  ========  ==========
  Step    Nref    RMSD_X    RMSD_Y    RMSD_Phi
                    (mm)      (mm)       (deg)
======  ======  ========  ========  ==========
     0   89562  0.046773  0.043273    0.021281
     1   89562  0.046872  0.042125    0.021623
     2   89562  0.04684   0.042104    0.021646
     3   89562  0.046778  0.042111    0.021662
     4   89562  0.046719  0.042129    0.02167
     5   89562  0.04666   0.042168    0.02167
     6   89562  0.046622  0.042203    0.021666
     7   89562  0.04661   0.042215    0.021664
     8   89562  0.046609  0.042216    0.021664
======  ======  ========  ========  ==========
simple
  Step    Nref    RMSD_X    RMSD_Y    RMSD_Phi
                    (mm)      (mm)       (deg)
------  ------  --------  --------  ----------
     0   89562  0.046773  0.043273    0.021281
     1   89562  0.046872  0.042125    0.021623
     2   89562  0.04684   0.042104    0.021646
     3   89562  0.046778  0.042111    0.021662
     4   89562  0.046719  0.042129    0.02167
     5   89562  0.04666   0.042168    0.02167
     6   89562  0.046622  0.042203    0.021666
     7   89562  0.04661   0.042215    0.021664
     8   89562  0.046609  0.042216    0.021664
textile
|_.    Step |_.   Nref |_.          RMSD_X
(mm) |_.          RMSD_Y
(mm) |_.          RMSD_Phi
(deg) |
|>.      0  |>.  89562 |>. 0.046773 |>. 0.043273 |>. 0.021281 |
|>.      1  |>.  89562 |>. 0.046872 |>. 0.042125 |>. 0.021623 |
|>.      2  |>.  89562 |>. 0.04684  |>. 0.042104 |>. 0.021646 |
|>.      3  |>.  89562 |>. 0.046778 |>. 0.042111 |>. 0.021662 |
|>.      4  |>.  89562 |>. 0.046719 |>. 0.042129 |>. 0.02167  |
|>.      5  |>.  89562 |>. 0.04666  |>. 0.042168 |>. 0.02167  |
|>.      6  |>.  89562 |>. 0.046622 |>. 0.042203 |>. 0.021666 |
|>.      7  |>.  89562 |>. 0.04661  |>. 0.042215 |>. 0.021664 |
|>.      8  |>.  89562 |>. 0.046609 |>. 0.042216 |>. 0.021664 |
tsv
  Step	  Nref	         RMSD_X
(mm)	         RMSD_Y
(mm)	         RMSD_Phi
(deg)
     0	 89562	0.046773	0.043273	0.021281
     1	 89562	0.046872	0.042125	0.021623
     2	 89562	0.04684 	0.042104	0.021646
     3	 89562	0.046778	0.042111	0.021662
     4	 89562	0.046719	0.042129	0.02167
     5	 89562	0.04666 	0.042168	0.02167
     6	 89562	0.046622	0.042203	0.021666
     7	 89562	0.04661 	0.042215	0.021664
     8	 89562	0.046609	0.042216	0.021664
youtrack
||    Step  ||    Nref  ||           RMSD_X
(mm)  ||           RMSD_Y
(mm)  ||           RMSD_Phi
(deg)  ||
|       0  |   89562  |  0.046773  |  0.043273  |  0.021281  |
|       1  |   89562  |  0.046872  |  0.042125  |  0.021623  |
|       2  |   89562  |  0.04684   |  0.042104  |  0.021646  |
|       3  |   89562  |  0.046778  |  0.042111  |  0.021662  |
|       4  |   89562  |  0.046719  |  0.042129  |  0.02167   |
|       5  |   89562  |  0.04666   |  0.042168  |  0.02167   |
|       6  |   89562  |  0.046622  |  0.042203  |  0.021666  |
|       7  |   89562  |  0.04661   |  0.042215  |  0.021664  |
|       8  |   89562  |  0.046609  |  0.042216  |  0.021664  |
```

### CODE REVIEW

**Comment by Anthchirp** [comment]
If you suggest that we want to change the table layout at some point then we should have a central place to do this. Specifically I would add to `/dials/util/__init__.py`:
```python
import functools
import tabulate as _tabulate

tabulate = functools.partial(_tabulate.tabulate, tablefmt="github")
```
and then use `from dials.util import tabulate` in the main code.

(incidentally this also selects the correct and best format by default)

**Comment by dagewa** [comment]
It looks like there are some formatting issues
```
Refinement steps:
  Step    Nref           RMSD_X
(mm)          RMSD_Y
(mm)           RMSD_Phi
(deg)
------  ------  --------  -------  --------
     0   17777  0.61951   0.35133  0.39852
     1   17777  0.1467    0.12843  0.19445
     2   17777  0.096138  0.11778  0.094816
     3   17777  0.07817   0.109    0.073743
     4   17777  0.075535  0.10823  0.072006
     5   17777  0.075403  0.10828  0.07191
     6   17777  0.075397  0.10829  0.071907
RMSD no longer decreasing

```

**Comment by dagewa** [comment]
That was with `github` format. `psql` is okay, and is quite similar to `simple_table`
```
Refinement steps:
+--------+--------+----------+----------+------------+
|   Step |   Nref |   RMSD_X |   RMSD_Y |   RMSD_Phi |
|        |        |     (mm) |     (mm) |      (deg) |
|--------+--------+----------+----------+------------|
|      0 |  17777 | 0.61951  |  0.35133 |   0.39852  |
|      1 |  17777 | 0.1467   |  0.12843 |   0.19445  |
|      2 |  17777 | 0.096138 |  0.11778 |   0.094816 |
|      3 |  17777 | 0.07817  |  0.109   |   0.073743 |
|      4 |  17777 | 0.075535 |  0.10823 |   0.072006 |
|      5 |  17777 | 0.075403 |  0.10828 |   0.07191  |
|      6 |  17777 | 0.075397 |  0.10829 |   0.071907 |
+--------+--------+----------+----------+------------+
```

**Comment by rjgildea** [comment]
It looks like tabulate doesn't support multiline cells for `tablefmt=github`:
https://github.com/astanin/python-tabulate/blob/19da809040c2f55b657c514909573247ef1cb86f/tabulate.py#L475-L489

I suspect that is because GitHub itself doesn't appear to support multiline cells, specifically it doesn't allow more than one row in the header. I.e. the following will render:

```
| item   | qty   |
|--------|-------|
| eggs   | 451   |
| more   | 42    |
| spam   |       |
```
| item   | qty   |
|--------|-------|
| eggs   | 451   |
| more   | 42    |
| spam   |       |

But the following will not render:
```
| item   | qty   |
| name   |       |
|--------|-------|
| eggs   | 451   |
| more   | 42    |
| spam   |       |
```
| item   | qty   |
| name   |       |
|--------|-------|
| eggs   | 451   |
| more   | 42    |
| spam   |       |

**Comment by rjgildea** [comment]
@dagewa I think I have to agree that `psql` is probably the best, or alternatively `orgtbl` which is similar, but does away with the top and bottom lines.
`tablefmt="orgtbl"`:
```
Refinement steps:
|   Step |   Nref |   RMSD_X |   RMSD_Y |   RMSD_Phi |
|        |        |     (mm) |     (mm) |      (deg) |
|--------+--------+----------+----------+------------|
|      0 |  89562 | 0.046773 | 0.043273 |   0.021281 |
|      1 |  89562 | 0.046872 | 0.042125 |   0.021623 |
|      2 |  89562 | 0.04684  | 0.042104 |   0.021646 |
|      3 |  89562 | 0.046778 | 0.042111 |   0.021662 |
|      4 |  89562 | 0.046719 | 0.042129 |   0.02167  |
|      5 |  89562 | 0.04666  | 0.042168 |   0.02167  |
|      6 |  89562 | 0.046622 | 0.042203 |   0.021666 |
|      7 |  89562 | 0.04661  | 0.042215 |   0.021664 |
|      8 |  89562 | 0.046609 | 0.042216 |   0.021664 |
```
vs. 
`tablefmt="psql"`:
```
Refinement steps:
+--------+--------+----------+----------+------------+
|   Step |   Nref |   RMSD_X |   RMSD_Y |   RMSD_Phi |
|        |        |     (mm) |     (mm) |      (deg) |
|--------+--------+----------+----------+------------|
|      0 |  89909 | 0.046608 | 0.042296 |   0.021701 |
|      1 |  89909 | 0.046134 | 0.040068 |   0.021746 |
|      2 |  89909 | 0.046112 | 0.040002 |   0.021763 |
|      3 |  89909 | 0.046109 | 0.039929 |   0.021727 |
|      4 |  89909 | 0.046116 | 0.039889 |   0.021556 |
|      5 |  89909 | 0.046119 | 0.039878 |   0.021326 |
|      6 |  89909 | 0.046119 | 0.039878 |   0.021223 |
|      7 |  89909 | 0.046119 | 0.039878 |   0.021212 |
|      8 |  89909 | 0.046119 | 0.039878 |   0.021212 |
+--------+--------+----------+----------+------------+
```

File: algorithms/refinement/reflection_manager.py [hunk 1]
```diff
@@ -731,10 +731,9 @@ def print_stats_on_matches(self):
         rows.append(["Y weights"] + ["%.4g" % e for e in row_data])
         row_data = five_number_summary(w_phi)
         rows.append(["Phi weights"] + ["%.4g" % (e * DEG2RAD ** 2) for e in row_data])
-        st = simple_table(rows, header)
 
         logger.info(msg)
-        logger.info(st.format())
+        logger.info(tabulate(rows, header))
         logger.info("")
```
Comment (Anthchirp) [inline algorithms/refinement/reflection_manager.py]: can this be concatenated into a single logger.info call?

File: algorithms/refinement/reflection_manager.py [hunk 2]
```diff
@@ -832,6 +830,5 @@ def print_stats_on_matches(self):
             + " matched to predictions:"
         )
         logger.info(msg)
-        st = simple_table(rows, header)
-        logger.info(st.format())
+        logger.info(tabulate(rows, header))
         logger.info("")
```
Comment (Anthchirp) [inline algorithms/refinement/reflection_manager.py]: can this be concatenated into a single logger.info call?

File: algorithms/refinement/two_theta_refiner.py [hunk 3]
```diff
@@ -97,9 +97,8 @@ def print_stats_on_matches(self):
         rows.append(
             ["2theta weights"] + ["%.4g" % (e * DEG2RAD ** 2) for e in row_data]
         )
-        st = simple_table(rows, header)
         logger.info(msg)
-        logger.info(st.format())
+        logger.info(tabulate(rows, header))
         logger.info("")
```
Comment (Anthchirp) [inline algorithms/refinement/two_theta_refiner.py]: can this be concatenated into a single logger.info call?

**Comment by ndevenish** [comment]
I'd argue for unicode tables but am probably alone in that! But this is a very good replacement!

**Pull request merged**